dist/21-526EZ-schema.json

```diff
@@ -1441,6 +1441,10 @@
             "Philippine Islands",
             "U.S. Minor Outlying Islands"
           ]
+        },
+        "zipCode": {
```

Comment:
Would it be of interest to create a definition for zipCode and reuse it in the various addresses, perhaps in a separate ticket?